app/models/manageiq/providers/azure/cloud_manager/provision/options_helper.rb

```diff
@@ -6,4 +6,8 @@ def root_password
   def resource_group
     @resource_group ||= ResourceGroup.find_by(:id => options[:resource_group])
   end
+
+  def security_group
+    @security_group ||= SecurityGroup.find_by(:id => get_option(:security_groups))
```

Comment:
Thanks for pointing this out in the description.  The UI is displaying a drop-down in the UI which is the wrong control for the other providers.  When that gets corrected it will be wrong for Azure, so we will likely need a few flag on the field to enforce single selection.

cc @dclarizio 

Screenshot from Version 5.4 showing the proper multi-select list for security groups:
![image](https://cloud.githubusercontent.com/assets/2591569/12983390/1413139c-d0b7-11e5-9a14-2c716e102fd4.png)